src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,99 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int ZERO = 0;
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 120;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkUser(user);
+        checkAgeNull(user);
+        checkAgeLess18(user);
+        checkAgeBigger100(user);
+        checkAgeZero(user);
+        checkLoginNull(user);
+        checkLoginIncorrect(user);
+        checkPasswordNull(user);
+        checkPasswordLess8(user);
+        checkIdNull(user);
+        checkIdNegative(user);
```

Comment:
you don't need to check the id, cause user gets the id only after `storageDao.add(user);`